service/src/main/java/bio/terra/externalcreds/dataAccess/EncryptDecryptUtils.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.externalcreds.dataAccess;
+
+import com.google.cloud.kms.v1.CryptoKeyName;
+import com.google.cloud.kms.v1.DecryptResponse;
+import com.google.cloud.kms.v1.EncryptResponse;
+import com.google.cloud.kms.v1.KeyManagementServiceClient;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+
+/** Utils for KMS symmetric encryption and decryption. */
+public class EncryptDecryptUtils {
+
+  private EncryptDecryptUtils() {}
+
+  /** Encrypt with KMS symmetric key. */
+  public static String encryptSymmetric(
```

Comment:
I would prefer to see this as a spring bean instead of static functions. That makes overriding and mocking much easier.